### DISCUSSION TITLE
Device

### DISCUSSION THREAD

**Discussion by dt-r** [hl7au/au-fhir-core#17] [AU Core R1 Resource Priority List - CLOSED]
Upvote for inclusion of a Resource Profile of [Device](http://hl7.org/fhir/R4/device.html) Resource in AU Core Release 1.

Comment on the profile(s) needed for Device in AU Core.

AU Core Device profile is not proposed. Device is profiled by use case in AU Core - there is no identified utility to date an 'AU Base Device' or 'AU Core Device' given the significant differences between use cases. The shared features are defined by the [Device](http://hl7.org/fhir/R4/device.html) Resource. Two potential AU Core use cases are identified:
- AU Core System Device
- AU Core Implantable or Wearable Medical Device

AU Core System Device sets minimum expectations for a Device resource to record, search, and fetch basic identifying and other administrative information about an information system device. It is based on the core [Device](http://hl7.org/fhir/R4/device.html) resource and identifies the additional mandatory core elements, extensions, vocabularies and value sets that SHALL be present in the Device when conforming to this profile. It provides the floor for standards development for specific uses cases in an Australian context.

AU Core Implantable Medical Device sets minimum expectations for a Device resource to record, search, and fetch basic identifying and other administrative information about an implantable medical device. It is based on the core [Device](http://hl7.org/fhir/R4/device.html) resource and identifies the additional mandatory core elements, extensions, vocabularies and value sets that SHALL be present in the Device when conforming to this profile. It provides the floor for standards development for specific uses cases in an Australian context.

**Comment by Indefessus** [comment]
I may be missing something but the links above to 'Device' go to 'Related Person' (http://hl7.org/fhir/R4/relatedperson.html).

**Reply by GaryPascoe** [reply]
Same for me @Indefessus ... with a nuance - the third and fourth references take me to the FHIR resource for 'Device'. The first and second link to the 'Related Person'. @dt-r can we please correct?

**Reply by dt-r** [reply]
@GaryPascoe & @Indefessus - fixed links.

**Comment by NickFerris** [comment]
Implantable medical device would seem more relevant to use cases in e-requesting, and in more general clinical applications
